logback-android/src/main/java/ch/qos/logback/core/android/AndroidContextUtil.java

```diff
@@ -60,7 +60,10 @@ public void setupProperties(LoggerContext context) {
     }
     props.setProperty(CoreConstants.PACKAGE_NAME_KEY, getPackageName());
     props.setProperty(CoreConstants.VERSION_CODE_KEY, getVersionCode());
-    props.setProperty(CoreConstants.VERSION_NAME_KEY, getVersionName());
+    final String versionName = getVersionName();
```

Comment:
IMO, `getVersionName()` should always return a string (and never `null`). Can you update the PR to do that instead?

Reply:
OK.  I'll make the change.